### PR TITLE
DTSPO-5506 update chart source

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml [hunk 1]
```diff
@@ -7,9 +7,14 @@ metadata:
 spec:
   releaseName: labs-mokainosspringboot
   chart:
-    git: git@github.com:hmcts/labs-mokainosspringboot
-    ref: master
-    path: charts/labs-mokainosspringboot
+    spec:
+      chart: ./charts/labs-mokainosspringboot
+      version: '4.0.x'
```
Comment (timja) [inline apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml:12]: is this right?

Comment (timja) [reply apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml:12]: I think it should be:

```
      sourceRef:
        kind: GitRepository
        name: hmcts-charts
        namespace: flux-system
```

Comment (mokainos) [reply apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml]: I have create a new gitrepository source

Comment (timja) [reply apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml]: shouldn't you use the hmcts-charts one?

File: apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml [hunk 2]
```diff
@@ -7,9 +7,13 @@ metadata:
...
   chart:
-    git: git@github.com:hmcts/labs-mokainosspringboot
-    ref: master
-    path: charts/labs-mokainosspringboot
+    spec:
+      chart: ./charts/labs-mokainosspringboot
+      sourceRef:
+        kind: GitRepository
+        name: labs-mokainosspringboot
+        namespace: labs
```
Comment (timja) [inline apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml:15]: should be this afaik
```suggestion
      sourceRef:
        kind: GitRepository
        name: hmcts-charts
        namespace: flux-system
```

**Review by timja — APPROVED** [review]

**Pull request merged**